app/cdap/components/GridTable/index.tsx

```diff
@@ -84,6 +82,8 @@ export default function GridTable() {
     },
   ]);
   const [snackbarState, setSnackbar] = useSnackbar();
+  const [recipeDetails, setRecipeDetails] = useState<IRecipeItem>();
+  const [recipeDetailIsOpen, setRecipeDetailsIsOpen] = useState(false);
```

Comment:
rename this to isRecipeDetailsPanelOpen

Reply:
Removed this variable from here as not needed